x/attribute/client/cli/cli_test.go

```diff
@@ -591,6 +593,83 @@ func (s *IntegrationTestSuite) TestAttributeTxCommands() {
 	}
 }
 
+func (s *IntegrationTestSuite) TestAttributeTxCommands_WithExpiration() {
+
+	testCases := []struct {
+		name            string
+		cmd             *cobra.Command
+		args            []string
+		expectErr       bool
+		respType        proto.Message
+		expectedCode    uint32
+		expectExpRecord bool
+	}{
+		{
+			"bind a new attribute name for testing",
+			namecli.GetBindNameCmd(),
+			[]string{
```

Comment:
Please use named parameters for these definitions. It makes them much easier to read and maintain.